### PR TITLE
HHOPPDEV-19267: snippets & highlighting

### PR DESCRIPTION
**Pull request by hexagon06**

### CODE REVIEW

File: syntaxes/omt.tmLanguage.json [hunk 1]
```diff
@@ -283,6 +283,28 @@
                 }
             ]
         },
+        "_merge-lists-def": {
```
Comment (emielb) [inline syntaxes/omt.tmLanguage.json:286]: Please put this above the `_merge-predicates-def` for keeping alphabetic order

File: syntaxes/omt.tmLanguage.json [hunk 2]
```diff
@@ -283,6 +283,28 @@
...
+                    "begin": "(?:\\G|^) +(subjects|predicates|when): +",
+                    "beginCaptures": {
+                        "1": { "name": "keyword.control.omt" }
+                    },
+                    "end": "$",
+                    "contentName": "meta.embedded.inline.odt",
+                    "patterns": [{ "include": "source.odt" }]
+                },
+                {
+                    "match": "(?:\\G|^) +(from): *(parent|both)?(.*)$",
```
Comment (emielb) [inline syntaxes/omt.tmLanguage.json:299]: The options `parent` and `both` are recognised here, but effectively all `!MergeLists` handlers act as though they're given the option `both`.

File: syntaxes/omt.tmLanguage.json [hunk 3]
```diff
@@ -937,6 +959,15 @@
                     },
                     "contentName": "meta.handler.validation.omt",
                     "patterns": [{ "include": "#_merge-validation-def" }]
+                },
+                {
```
Comment (emielb) [inline syntaxes/omt.tmLanguage.json:963]: Please move this up above `!MergePredicates`

**Review by emielb — CHANGES_REQUESTED** [review]
Mostly sorting

File: syntaxes/omt.tmLanguage.json (hunk 2 above)
Comment (hexagon06) [reply syntaxes/omt.tmLanguage.json:299]: Implement differences between options. then this can be resolved.

Comment (emielb) [reply syntaxes/omt.tmLanguage.json]: This is fixed now, but I guess the CHANGELOG.md date needs an update and then we can merge :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,14 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations
 on how to structure this file.
 This plugin uses [Semantic versioning](https://semver.org).
 
+## [2.8.0] - 2021-04-26
```
Comment (emielb) [inline CHANGELOG.md:10]:
```suggestion
## [2.8.0] - 2021-05-04
```

**Review by emielb — APPROVED** [review]

**Review by yasinkocak — APPROVED** [review]
LGTM

**Pull request merged**